### PR TITLE
Fix for drive.google.com

### PR DESCRIPTION
**Pull request by rakleed**
- Resolves https://github.com/darkreader/darkreader/issues/5924.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -3433,7 +3433,7 @@ drive.google.com
 INVERT
 img[src$="google_gsuite"]
 div[role="menuitem"] svg
-div[role="dialog"] div[role="button"][tabindex="0"]
+div[role="dialog"][style^="width: 350px;"] div[role="button"][tabindex="0"]
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:3436]: Does this play well with different screen resolutions?

Comment (rakleed) [reply src/config/dynamic-theme-fixes.config:3436]: Yes, I checked.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:3436]: Great. Il take a look to fix the other problem that was mentioned.

**Comment by Gusted** [comment]
@rakleed 

```CSS
div[role="dialog"] ~ div[role="menu"] > div[role="menuitem"] > div > div:not([style*="background-image"])
```

It matches if any dialog is displayed(like pdf preview) then it will `generallty select the role=menu sibling`(https://developer.mozilla.org/en-US/docs/Web/CSS/General_sibling_combinator/General_sibling_combinator) and then it has the same fix as drive.google.com/file I've tested it for 10 minutes across various pages and it should for in this exact case and any other dialog's that has this toolbar(hopefully).

**Review by Gusted — APPROVED** [review]

**Pull request merged**